backend/migrate_cycle_count.py

```
@@ -4,140 +1,149 @@
+    try:
+        with app.app_context():
+            logger.info("Verifying cycle count functionality...")
+            
+            # Test basic queries
+            schedules_count = CycleCountSchedule.query.count()
+            batches_count = CycleCountBatch.query.count()
+            items_count = CycleCountItem.query.count()
+            results_count = CycleCountResult.query.count()
+            adjustments_count = CycleCountAdjustment.query.count()
+            
+            logger.info(f"Current cycle count data:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)